modules/system-view/src/main/java/org/apache/ignite/internal/systemview/utils/SystemViewUtils.java

```diff
@@ -66,13 +70,14 @@ public static CreateSystemViewCommand toSystemViewCreateCommand(SystemView<?> vi
     }
 
     private static ColumnParams systemViewColumnToColumnParams(SystemViewColumn<?, ?> column) {
-        NativeTypeSpec typeSpec = NativeTypeSpec.fromClass(column.type());
+        NativeType type = column.type();
+
+        NativeTypeSpec typeSpec = type.spec();
 
         Builder builder = ColumnParams.builder()
                 .name(column.name())
```

Comment:
And maybe we should set nullability of standard system view columns to `false`?

Reply:
good catch! fixed